Mathlib/Algebra/Star/Unitary.lean

```diff
@@ -135,12 +136,36 @@ theorem _root_.IsUnit.mem_unitary_iff_mul_star_self {u : R} (hu : IsUnit u) :
 alias ⟨_, _root_.IsUnit.mem_unitary_of_star_mul_self⟩ := IsUnit.mem_unitary_iff_star_mul_self
 alias ⟨_, _root_.IsUnit.mem_unitary_of_mul_star_self⟩ := IsUnit.mem_unitary_iff_mul_star_self
 
+lemma mul_inv_mem_unitary {G : Type*} [Group G] [StarMul G] (a b : G) :
```

Comment:
```suggestion
lemma mul_inv_mem_unitary_iff {G : Type*} [Group G] [StarMul G] (a b : G) :
```
throughout. I think you might be in a weird namespace here, so perhaps even
```suggestion
lemma mul_inv_mem_iff {G : Type*} [Group G] [StarMul G] (a b : G) :
```
or
```suggestion
lemma _root_.mul_inv_mem_unitary {G : Type*} [Group G] [StarMul G] (a b : G) :
```